test/cmd/newapp.sh

```diff
@@ -18,6 +18,8 @@ os::cmd::expect_failure 'oc new-app unknownhubimage -o yaml'
 os::cmd::expect_failure_and_text 'oc new-app docker.io/node~https://github.com/openshift/nodejs-ex' 'the image match \"docker.io/node\" for source repository \"https://github.com/openshift/nodejs-ex\" does not appear to be a source-to-image builder.'
 os::cmd::expect_failure_and_text 'oc new-app https://github.com/openshift/rails-ex' 'the image match \"ruby\" for source repository \"https://github.com/openshift/rails-ex\" does not appear to be a source-to-image builder.'
 os::cmd::expect_success 'oc new-app https://github.com/openshift/rails-ex --strategy=source --dry-run'
+os::cmd::expect_success 'oc new-app  openshift/postgresql-92-centos7 --env-file=/tmp/envfile'
```

Comment:
it is not related to newapp_test.go.  This shell script is what we'd call an "end to end" test, it exercises the full code as a user would.  newapp_test.go is a unit test, it exercises specific subsets of functionality by calling specific functions directly.

Adding this line is fine, but you need to:
1) create /tmp/envfile (and clean it up after)
2) actually confirm via the output text that new-app included the env vars from /tmp/envfile (see: expect_success_and_text examples)
3) you can also add expect_failure_and_text tests, for cases where the --env-file does not exist, or contains invalid content.

and if you add all those things here, you do not need to add a test in newapp_test.go, it's really either or.